### ISSUE TITLE
Is our preview screen accessible?

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#673]
This was an older issue that I found in my notes:

Our preview screen makes folks download potentially non-accessible PDFs to verify that they answered questions as expected. A2J author added a feature to accessibly read our someone's answers so far, we should be able to do a similar thing in docassemble.

some remedies to the fact that downloading PDFs are not accessible
• say on screen that it's not accessible
• have review screen on same screen, or button still depending on the length
• need to manual test to see if it's possible to improves for some forms without recreating all of them from scratch.

**Comment by nonprofittechy** [comment]
#490 could fix this. I think it properly belongs to the ALWeaver though, not here.
